test/functional/cmdLineTests/shareClassTests/SCCommandLineOptionTests/playlist.xml

```diff
@@ -50,5 +50,40 @@
 			<impl>openj9</impl>
 			<impl>ibm</impl>
 		</impls>
+		<versions>
+			<version>8</version>
+		</versions>
+	</test>
+	<test>
+		<testCaseName>cmdLineTester_SCCommandLineOptionTests_SE110</testCaseName>
+		<variations>
+			<variation>Mode110</variation>
+			<variation>Mode610</variation>
+		</variations>
+		<command>cp $(Q)$(TEST_RESROOT)$(D)SCCommandLineOptionTests.jar$(Q) .; \
+	$(Q)$(TEST_JDK_HOME)$(D)bin$(D)jar$(EXECUTABLE_SUFFIX)$(Q) xf SCCommandLineOptionTests.jar; \
+	$(CONVERT_TO_EBCDIC_CMD) \
+	$(JAVA_COMMAND) $(JVM_OPTIONS) -DCPDL=$(Q)$(P)$(Q) -DRUN_SCRIPT=$(RUN_SCRIPT) -DSCRIPT_SUFFIX=$(SCRIPT_SUFFIX) -DEXECUTABLE_SUFFIX=$(EXECUTABLE_SUFFIX) -DTEST_JDK_HOME=$(Q)$(TEST_JDK_HOME)$(Q) \
+	-jar $(CMDLINETESTER_JAR) \
+	-config $(Q)$(TEST_RESROOT)$(D)CommandLineOptionTests.xml$(Q) \
+	-xids all,$(PLATFORM),$(VARIATION) -xlist $(Q)$(TEST_RESROOT)$(D)exclude_$(JDK_VERSION).xml$(Q) \
```

Comment:
By having `exclude_$(JDK_VERSION).xml`, we have to create exclude file for each java version >=11. I think a simple way may be making this a variable in make file.

```
EXCLUDE_FILE=exclude_11.xml
ifeq (8, $(JDK_VERSION))
 EXCLUDE_FILE=exclude.xml
endif
```

In playlist, `-xids all,$(PLATFORM),$(VARIATION) -xlist $(Q)$(TEST_RESROOT)$(D)$(EXCLUDE_FILE)$(Q)` in the original test target.

Similar example: https://github.com/eclipse-openj9/openj9/blob/6d3839791d7e385c57d9da895f4b8751706545da/test/functional/cmdLineTests/shareClassTests/StoreFilterTests/variables.mk

Reply:
Updated. Grinder passed: https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/17896/